www/include/configuration/configObject/traps/listTraps.php

```diff
@@ -53,15 +53,20 @@
     FILTER_SANITIZE_STRING
 );
 
-$searchStatus = filter_var(
-    $_POST['status'] ?? $_GET['status'] ?? null,
-    FILTER_SANITIZE_NUMBER_INT
-);
+if ($search != '') {
+    $searchStatus = filter_var(
+        $_POST['status'] ?? $_GET['status'] ?? null,
+        FILTER_SANITIZE_NUMBER_INT
+    );
 
-$searchVendor = filter_var(
-    $_POST['vendor'] ?? $_GET['vendor'] ?? null,
-    FILTER_SANITIZE_NUMBER_INT
-);
+    $searchVendor = filter_var(
+        $_POST['vendor'] ?? $_GET['vendor'] ?? null,
+        FILTER_SANITIZE_NUMBER_INT
+    );
+} else {
+    $searchStatus = null;
+    $searchVendor = null;
+}
 
 if (isset($_POST['searchT']) || isset($_GET['searchT'])) {
```

Comment:
same comment. If the customer wich to select only a vendor, shouldn't we save its presets ?

Reply:
maybe you could concatenate this block with the next (line 84) by replacing the line 71 with the line 84 condition